### ISSUE TITLE
The accuracy of the pre-trained model does not match the paper?

### ISSUE THREAD

**Issue by why529913** [JOP-Lee/READ#64]
I use --headless, test with the pre-trained model on kitt6 dataset (image_kitti6_all_368.zip), and find that the accuracy does not match, I want to know which scene which dataset the pre-trained model is, whether to use headless, I want to reproduce the accuracy as much as possible. thanks.

**Comment by lhc991025** [comment]
This is the model trained by myself
![image](https://github.com/user-attachments/assets/6b87f279-63db-4c72-8e39-035fa89514e7)
Why is the author's pre trained model better, even better than what is written in the paper. Is my metric method written incorrectly?
![image](https://github.com/user-attachments/assets/aedf8fcb-7412-4d95-a1e5-72d029d403d1)
